muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/workspaces/index.ts

```diff
@@ -1,22 +1,20 @@
 import actions, { displayNotification } from '../base/notifications';
 import promisify from '~/util/promisify';
 import mApi, { MApiError } from '~/lib/mApi';
-import {AnyActionType, SpecificActionType} from '~/actions';
-import { WorkspaceListType, WorkspaceMaterialReferenceType, WorkspaceType, WorkspaceChatStatusType, WorkspaceStudentActivityType, WorkspaceStudentAssessmentsType, WorkspaceFeeInfoType, WorkspaceAssessementStateType, WorkspaceAssessmentRequestType, WorkspaceEducationFilterListType, WorkspaceCurriculumFilterListType, WorkspacesActiveFiltersType, WorkspacesStateType, WorkspacesPatchType, WorkspaceAdditionalInfoType, WorkspaceUpdateType} from '~/reducers/workspaces';
+import { WorkspaceListType, WorkspaceMaterialReferenceType, WorkspaceType, WorkspaceChatStatusType, WorkspaceStudentActivityType, WorkspaceStudentAssessmentsType, WorkspaceFeeInfoType, WorkspaceAssessementStateType, WorkspaceAssessmentRequestType, WorkspaceEducationFilterListType, WorkspaceCurriculumFilterListType, WorkspacesActiveFiltersType, WorkspacesStateType, WorkspacesPatchType, WorkspaceAdditionalInfoType, WorkspaceUpdateType } from '~/reducers/workspaces';
+import { AnyActionType, SpecificActionType } from '~/actions';
 import { StateType } from '~/reducers';
 import { loadWorkspacesHelper, loadCurrentWorkspaceJournalsHelper } from '~/actions/workspaces/helpers';
 import { UserStaffType, ShortWorkspaceUserWithActiveStatusType } from '~/reducers/user-index';
-import { MaterialContentNodeListType, MaterialCompositeRepliesListType, MaterialCompositeRepliesStateType,
+import {
+  MaterialContentNodeListType, MaterialCompositeRepliesListType, MaterialCompositeRepliesStateType,
   WorkspaceJournalsType, WorkspaceJournalType, WorkspaceDetailsType, WorkspaceTypeType, WorkspaceProducerType,
-  WorkspacePermissionsType, WorkspaceMaterialEditorType, MaterialContentNodeProducerType, MaterialContentNodeType,
-  WorkspaceEditModeStateType,
-  WorkspaceOrganizationFilterListType} from '~/reducers/workspaces';
+  WorkspacePermissionsType, WorkspaceMaterialEditorType, WorkspaceStateFilterListType, MaterialContentNodeProducerType, MaterialContentNodeType,
+  WorkspaceEditModeStateType, UserSelectLoader, WorkspaceOrganizationFilterListType
+} from '~/reducers/workspaces';
 import equals = require("deep-equal");
 import $ from '~/lib/jquery';
-
-export interface LoadUserWorkspacesFromServerTriggerType {
-  ():AnyActionType
-}
+import { SelectItem } from '~/components/base/input-select-autofill';
```

Comment:
Importing an UI specific element into the action is not a great idea, the action should be pure.

Reply:
does not exist here anymore, but github insists it does